tests/storage/cdi_import/utils.py

```
@@ -0,0 +1,100 @@
+
+    Raises:
+        TimeoutExpiredError: If the importer pod is not scheduled within the timeout period.
+    """
+    for sample in TimeoutSampler(
+        wait_timeout=TIMEOUT_1MIN,
+        sleep=TIMEOUT_5SEC,
+        func=lambda: importer_pod.instance["spec"].get("nodeName"),
+    ):
+        if sample:
+            return sample
+    raise TimeoutExpiredError("Importer pod was not scheduled within the timeout period.")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)